cmd/root.go

```diff
@@ -162,6 +173,31 @@ func getCacheDir(imageName string) (string, error) {
 	return filepath.Join(rootDir, filepath.Clean(imageName)), nil
 }
 
+func getWriter(outputFile string) (io.Writer, error) {
+	var err error
+	var outWriter io.Writer
+
+	// If the user specifies an output file, ensure exists
+	if outputFile != "" {
+
+		// Don't overwrite a file that exists, unless given --force
+		if _, err := os.Stat(outputFile); !os.IsNotExist(err) {
+			if !forceWrite {
+				logrus.Error("file exist, will not overwrite.")
```

Comment:
usually we try to send the error back up as high in the call stack as we can, and then log it up there. so instead of logging this here, can you just return `errors.Wrap(err, ...)` or just `errors.New(...)`?

Reply:
Definitely.